src/app/containers/Ad/Amp/index.test.jsx

```diff
@@ -55,6 +64,14 @@ describe('AMP Ads', () => {
       });
     });
 
+    it('should render an `advertisement` label', () => {
```

Comment:
Another way we can write this test is like this:
```
  const { getByText } = renderAmpAd();
  const  adsLabel = getByText('Tori we dem pay for');

  expect(adsLabel).toBeInTheDocument();
```

^ this way we aren't explicitly searching through the element tags (saves us from updating this test if were to change the element to a <a> )

Reply:
This would check the whole component for any elements with the words `Tori we dem pay for` in it. It would be hard to assert if it's a `link` or just a `paragraph`. 

The current implementation explicitly checks that it's a `paragraph` and that it contains the desired text (`'Tori we dem pay for'`)